setup.py

```diff
@@ -92,15 +92,15 @@ def run(self):
     install_requires=REQUIRED,
     extras_require={"dev": DEV_REQUIRED},
     include_package_data=True,
-    license="MIT",
+    license="Apache 2.0",
     keywords="airflow",
     classifiers=[
         # Trove classifiers
         # Full list: https://pypi.python.org/pypi?%3Aaction=list_classifiers
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.8",
```

Comment:
Since it is >3.8

We could add more versions too

Reply:
For now, I have added Python3.{8,9,10}. I'll add 3.11, 3.12 once will start running tests for it